docs/source/guide/webhook_reference.md

```diff
@@ -0,0 +1,209 @@
+
+## Task Created
+
+Sent when a task is created in Label Studio. See how to [set up a webhook for this event](webhooks.html). 
+
+### Webhook payload details
+
+The webhook payload includes the name of the action and some additional task data. The task-relevant data is the same as is included in the [response when you create a task using the API](/api#operation/api_tasks_create).
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string | Name of the action. In this case, `Task created` | 
+| id | integer | ID of the created task. | 
+| data | | Reference to the data associated with the task. Can be a URL such as `s3://path/to/bucket/image.png` |
+| meta | | If it exists, metadata about the task. |
+| project | integer | Project ID for the task. |
+| created_at | datetime | Date and time of task creation. |
+| updated_at | datetime | Date and time of last update to the task. |
+| is_labeled | boolean | Whether or not the task has been labeled. |
+
+### Example payload
+
+```json
+{
+    "action": "Task created",
+    "tasks": [
+        {"id": 1, ...},
+        ...
+    ]
+}
+```
+
+## Task Deleted
+
+Sent when a task is deleted from Label Studio. See how to [set up a webhook for this event](webhooks.html)
+
+### Webhook payload details
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string | Name of the action. In this case, `Task deleted`. | 
+| id | integer | ID of the deleted task. | 
+
+### Example payload 
+
+```json
+{
+    "action": "Task deleted",
+    "tasks": [
+        {"id": 1, ...},
+        ...
+    ]
+}
+```
+
+## Annotation Created
+Sent when an annotation is created for a task in Label Studio. See how to [set up a webhook for this event](webhooks.html)
+
+### Webhook payload details
+
+The webhook payload includes the name of the action and some additional annotation data. The annotation-relevant data is the same as is included in the [response when you create an annotation using the API](/api#operation/api_tasks_annotations_create).
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string | Name of the action. In this case, `Annotation created`. | 
+| id | integer | ID of the created annotation. | 
+| result | JSON | JSON format of the annotation created. |
+| task | JSON | The task that the annotation was created for. |
+| completed_by | JSON | ID of the user that created the annotation. |
+| was_cancelled | boolean | Whether or not the annotation is the result of a skipped task, and an empty annotation. |
+| ground_truth | boolean | Enterprise only. Whether or not the annotation is a ground truth. 
+| created_at | datetime | Date and time that the annotation was created. |
+| updated_at | datetime | Date and time that the annotation was last updated. |
+| prediction | JSON dictionary | Details of the prediction viewed at the time of annotation, if one exists. |
```

Comment:
need to ref link this to prediction JSON payload

Reply:
hmm that's also missing from the API reference so I'm not sure what format it would take